### DISCUSSION TITLE
🚀 FR Ability to unselect radio field (and have unselected by default)

### DISCUSSION THREAD

**Discussion by robinsowell** [ExpressionEngine/ExpressionEngine#4085] [Feature Requests]
### Suggest an idea for ExpressionEngine

User wants to have a set of radio buttons with no option selected.  There's no way to do it and really no reasonable way to unselect a radio.  

So this is a request for both options to be added- a way to set the default behavior to NOT have the first radio selected and a way to unselect a radio to get back to a null state.

See also the larger discussion of defaults for select type fields: https://github.com/ExpressionEngine/ExpressionEngine/discussions/3214

I will say we shouldn't change what's happening with radios now- they have always defaulted to having the first option selected.  But giving the ability to set a 'no selected' option in settings would be nice.  As would a way to deselect an existing radio.

**Comment by intoeetive** [comment]
Well, the radio buttons in HTML are build in the way that you can't deselect those. Unlike checkboxes

But - we're doing just a little bit more than simple radio, so may consider adding a config setting like "add empty row".

Would be also useful for select - often people need to a add blank row to options manually to enable deselecting
